apps/iglidur-designer/src/app/libs/header/src/lib/header/header.component.scss

```diff
@@ -1,73 +1,78 @@
 .header {
-    height: 3.5rem;
-    background-color: rgb(54, 54, 54);
+  height: 3.5rem;
+  background-color: rgb(54, 54, 54);
+  display: flex;
```

Comment:
I see you are using a lot of this flex justify align.
It would make sense to create a mixin for it

Reply:
I have refactored the code to use mixins